plugin/evm/config/config.go

```diff
@@ -224,65 +158,6 @@ type Config struct {
 	HttpBodyLimit uint64 `json:"http-body-limit"`
 }
 
-// EthAPIs returns an array of strings representing the Eth APIs that should be enabled
-func (c Config) EthAPIs() []string {
-	return c.EnabledEthAPIs
-}
-
-func (c Config) EthBackendSettings() eth.Settings {
-	return eth.Settings{MaxBlocksPerRequest: c.MaxBlocksPerRequest}
-}
```

Comment:
For this, since it's used in only one place (unless it's meant to be exported as Go API), you could do the operation in plugin/evm by changing

```go
	vm.eth, err = eth.New(
		node,
		&vm.ethConfig,
		&EthPushGossiper{vm: vm},
		vm.chaindb,
		vm.config.EthBackendSettings(),
		lastAcceptedHash,
		dummy.NewFakerWithClock(callbacks, &vm.clock),
		&vm.clock,
	)
```

to

```go
	vm.eth, err = eth.New(
		node,
		&vm.ethConfig,
		&EthPushGossiper{vm: vm},
		vm.chaindb,
		eth.Settings{MaxBlocksPerRequest: vm.config.MaxBlocksPerRequest},
		lastAcceptedHash,
		dummy.NewFakerWithClock(callbacks, &vm.clock),
		&vm.clock,
	)
```

I think 🤔